deployment/helm/values.yaml

```diff
@@ -16,6 +16,13 @@ useDevelopmentContainers: true
 # This can be set from the helm command line using `--set imagePullSecrets[0].name="mysecret"`
 imagePullSecrets: []
 
+# kubernetesDistro describes the Kubernetes distro Akri is running on. It is used to conditionally set
+# distribution specific values such as container runtime socket.
+kubernetesDistro:
+  microk8s: false
```

Comment:
i think i'd lean towards using kubernetesDistro==k8s | kubernetesDistro==microk8s | kubernetesDistro==k3s ... rather than kubernetesDistro.microk8s==true, etc.  

seems like maybe a cleaner way to make sure one and only one distro is ever specified.

Reply:
I was worried about spelling but this makes sense. Changed it all!